src/main/java/com/pirimid/cryptotrade/controller/OrderController.java

```diff
@@ -0,0 +1,50 @@
+package com.pirimid.cryptotrade.controller;
+
+import com.pirimid.cryptotrade.DTO.PlaceOrderReqDTO;
+import com.pirimid.cryptotrade.model.Order;
+import com.pirimid.cryptotrade.service.OrderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Set;
+import java.util.UUID;
+
+@RestController
+public class OrderController {
+    @Autowired
+    OrderService orderService;
+
+    @GetMapping("/orders/{orderId}")
+    public ResponseEntity<Order> getOrderById(@PathVariable UUID orderId){
+        Order order = orderService.getOrderById(orderId);
+        if(order==null) return ResponseEntity.badRequest().body(null);
+        return ResponseEntity.status(HttpStatus.ACCEPTED).body(order);
+    }
+//    @GetMapping("/orders") // account order
```

Comment:
Do we want to remove this comment if not require?

Reply:
yes